guides/source/active_record_querying.md

```diff
@@ -345,6 +345,7 @@ irb> Customer.find_by first_name: 'Lifo'
 irb> Customer.find_by first_name: 'Jon'
 => nil
 ```
+There is no implied ordering so if the order matters, you should specify it yourself.
```

Comment:
I feel like this line breaks the flow a bit.  What if we moved the caveat further down?  Something like:

~~~diff
diff --git a/guides/source/active_record_querying.md b/guides/source/active_record_querying.md
index 99259c25f2..ec62bd04d1 100644
--- a/guides/source/active_record_querying.md
+++ b/guides/source/active_record_querying.md
@@ -358,6 +358,8 @@ The SQL equivalent of the above is:
 SELECT * FROM customers WHERE (customers.first_name = 'Lifo') LIMIT 1
 ```
 
+Note that there is no `ORDER BY` in the above SQL.  If your `find_by` conditions can match multiple records, you should [apply an order](#ordering) to guarantee a deterministic result.
+
 The [`find_by!`][] method behaves exactly like `find_by`, except that it will raise `ActiveRecord::RecordNotFound` if no matching record is found. For example:
 
 ```irb
~~~

Reply:
This looks better. Have updated the PR @jonathanhefner